api/request.go

```diff
@@ -139,9 +139,9 @@ func (s SliceRequest) toString() (string, error) {
 	return str, nil
 }
 
-// Query for Horizon endpoints
-// @Description Query payload for horizon endpoint /horizon.
-type HorizonRequest struct {
+// Query for Attribute endpoints
+// @Description Query payload for attribute endpoint.
+type AttributeRequest struct {
```

Comment:
Also note that now user sees horizon endpoint under horizon tag, but is supposed to use AttributeRequest to create the request itself.
I think there is no link that Horizon endpoint relates to AttributeRequest in any way.

So not sure if AttributeRequest shouldn't be HorizonRequest or we should find another way to map them.

Reply:
Unsure, but the way the swagger docs are layed out I don't think this will be a source of much confusion